mava/wrappers/episode_metrics.py

```diff
@@ -72,13 +79,33 @@ def step(
         timestep.extras["episode_metrics"] = {
             "episode_return": episode_return_info,
             "episode_length": episode_length_info,
+            "is_terminal_step": done,
         }
 
         state = RecordEpisodeMetricsState(
             env_state=env_state,
+            key=state.key,
             running_count_episode_return=new_episode_return * not_done,
             running_count_episode_length=new_episode_length * not_done,
             episode_return=episode_return_info,
             episode_length=episode_length_info,
         )
         return state, timestep
+
+
+def get_final_step_metrics(metrics: Dict[str, chex.Array]) -> Dict[str, chex.Array]:
+    """Get the metrics for the final step of an episode.
+
+    Note: this is not a jittable method. We need to return variable length arrays, since
+    we don't know how many episodes have been run.
+    """
```

Comment:
```suggestion
    """Get the metrics for the final step of an episode.

    Note: this is not a jittable method. We need to return variable length arrays, since
    we don't know how many episodes have been run. This is done since the logger
    expects arrays for computing summary statistics on the episode metrics.
    """
```